sdk/ai/azure-ai-projects/samples/agents/async_samples/sample_agents_azure_functions_async.py

```diff
@@ -0,0 +1,110 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+import asyncio
+
+"""
+FILE: sample_agents_azure_functions_async.py
```

Comment:
Remove the "FILE: ... " lines from all samples, per my recent change

Reply:
Fixed